scripts/generate.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```

Comment:
let's inspire from rvm, nvm

imagine we call it svm # sonic version manager

`curl sonic.io/svm | bash`

clone repo + install bash function or bash script "svm"

then user can type in bash 

```
svm github.com/openware/appsonic

cp -r $HOME/.svm/skel appsonic
sed -i "s#github.com/openware/sonic/skel#$APP#g" appsonic/**/*.go
```